src/cljs/bluegenes/pages/querybuilder/events.cljs

```diff
@@ -355,6 +370,37 @@
                      :order new-order)
       :dispatch [:qb/enhance-query-build-im-query true]})))
 
+(defn subpath-of? [subpath path]
+  (let [subpaths (->> (split path #"\.")
+                      (iterate drop-last)
+                      (take-while not-empty)
+                      (set))]
+    (contains? subpaths (split subpath #"\."))))
```

Comment:
not a blocker for merge, but should this be in imcljs or better here? 🌷

Reply:
I thought long about the same... Can't remember what tipped me over to leave it out of imcljs.